xml/System.Globalization/CompareInfo.xml

```diff
@@ -256,12 +256,14 @@
         <Parameter Name="options" Type="System.Globalization.CompareOptions" Index="2" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="string1">To be added.</param>
-        <param name="string2">To be added.</param>
-        <param name="options">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="string1">The first string to compare.</param>
```

Comment:
>string [](start = 40, length = 6)

do we usually use the word "string" for Span<char>?

Reply:
I think I've seen string being used for spans, but I can correct it. How about this?:
```suggestion
        <param name="string1">The first read-only span of characters to compare.</param>
```